### PR TITLE
Updated product mappings for synthetic Diesel fuel types

### PR DESCRIPTION
**Pull request by jpwenzel**

### CODE REVIEW

File: 1.0/ext/product-mapping.md [hunk 1]
```diff
@@ -77,33 +77,46 @@ products_method = "PRODUCTS" .
...
+| `cng`                | Compressed natural gas                                                    |
+| `diesel`             | Diesel fuel                                                               |
+| `dieselB0`           | Diesel fuel (0% bio additives)                                            |
+| `dieselB7`           | Diesel fuel (up to 7% bio additives)                                      |
+| `dieselB15`          | Diesel fuel (up to 15% bio additives)                                     |
+| `dieselB20`          | Diesel fuel (up to 20% bio additives)                                     |
+| `dieselBMix`         | Diesel fuel (unspecified amount of additives)                             |
+| `dieselCare`         | C.A.R.E. Diesel (synthetic)                                               |
+| `dieselGtl`          | GTL/XTL Diesel (synthetic, gas-to-liquid or x-to-liquid)                  |
+| `dieselHvo100`       | HVO Diesel (100%)                                                         |
```
Comment (pieceofsoul) [inline 1.0/ext/product-mapping.md]: HVO is the abbreviation for `Hydrogenated Vegetable Oils` - I see not a reason to add a number indicating a mixing ratio / share. Instead I would propose to use `dieselHVO`

File: 1.0/ext/product-mapping.md [hunk 2]
```diff
@@ -77,33 +77,46 @@ products_method = "PRODUCTS" .
...
+| `adBlue`             | AdBlue Diesel exhaust fluid (DEF, also: AUS 32)                           |
+| `careDiesel` *       | => deprecated, use `dieselCare` instead                                   |
+| `cng`                | Compressed natural gas                                                    |
+| `diesel`             | Diesel fuel                                                               |
+| `dieselB0`           | Diesel fuel (0% bio additives)                                            |
+| `dieselB7`           | Diesel fuel (up to 7% bio additives)                                      |
+| `dieselB15`          | Diesel fuel (up to 15% bio additives)                                     |
+| `dieselB20`          | Diesel fuel (up to 20% bio additives)                                     |
+| `dieselBMix`         | Diesel fuel (unspecified amount of additives)                             |
+| `dieselCare`         | C.A.R.E. Diesel (synthetic)                                               |
```
Comment (pieceofsoul) [inline 1.0/ext/product-mapping.md]: According to what I can find on the net this is basically `dieselHVO`. I would propose to remove this completely.

File: 1.0/ext/product-mapping.md [hunk 3]
```diff
@@ -77,33 +77,46 @@ products_method = "PRODUCTS" .
...
+| `dieselGtl`          | GTL/XTL Diesel (synthetic, gas-to-liquid or x-to-liquid)                  |
+| `dieselHvo100`       | HVO Diesel (100%)                                                         |
+| `dieselPremium`      | Premium Diesel                                                            |
+| `dieselRed`          | Red/colored Diesel (used in agricultural vehicles or generators)          |
+| `dieselSynthetic`    | Other types of synthetic Diesel (EN 15940 fuel)                           |
```
Comment (pieceofsoul) [inline 1.0/ext/product-mapping.md:95]: If we add `dieselB100` as fully renewable fuel variant, I do not see currently a reason to add a generic synthetic fuel type.

File: 1.0/ext/product-mapping.md [hunk 4]
```diff
@@ -77,33 +77,46 @@ products_method = "PRODUCTS" .
...
+| `ron98`              | RON 98 grade gasoline                                                     |
+| `ron98e5`            | RON 98 grade gasoline (up to 5% bio additives)                            |
+| `ron100`             | RON 100 grade gasoline                                                    |
+| `syntheticDiesel` *  | => deprecated, use `dieselSynthetic` instead                              |
+| `truckAdBlue`        | AdBlue Diesel exhaust fluid (DEF, also: AUS 32) for commercial vehicles   |
+| `truckDiesel`        | Diesel for commercial vehicles (including agricultural diesel fuel)       |
+| `truckDieselPremium` | Premium Diesel for commercial vehicles                                    |
+| `truckLpg`           | Liquefied petroleum gas for commercial vehicles                           |
+
+Deprecated fuel types are marked with `*` in the list above. Do not use them for new implementations, they will be removed in next version:
```
Comment (pieceofsoul) [inline 1.0/ext/product-mapping.md]: `in an upcoming version`

File: 1.0/ext/product-mapping.md (hunk 2 above)
Comment (jpwenzel) [reply 1.0/ext/product-mapping.md]: Agreed.

File: 1.0/ext/product-mapping.md (hunk 3 above)
Comment (jpwenzel) [reply 1.0/ext/product-mapping.md:95]: I'd like to keep this as a potential fallback for now

File: 1.0/ext/product-mapping.md (hunk 4 above)
Comment (jpwenzel) [reply 1.0/ext/product-mapping.md]: ACK

**Pull request merged**